rustler_mix/lib/rustler.ex

```diff
@@ -18,34 +18,98 @@ defmodule Rustler do
 
   Configuration options:
 
+    * `:cargo` - Specify how to envoke the rust compiler. Options are:
+        - `:system` (default) - use `cargo` from the system (must by in `$PATH`)
+        - `{:rustup, <version>}` - use `rustup` to specify which channel to use.
+          Available options include: `:stable`, `:beta`, `:nightly`, or a string
+          which specifies a specific version (i.e. `"1.39.0"`).
+        - `{:bin, "/path/to/binary"}` - provide a specific path to `cargo`.
+
     * `:crate` - the name of the Rust crate (as an atom), if different from your
       `otp_app` value. If you have more than one crate in your project, you will
       need to be explicit about which crate you intend to use.
 
+    * `:default_features` - a boolean to specify whether the crate's default features
+      should be used.
+
+    * `:env` - Specify a list of environment variables when envoking the compiler.
+
+    * `:feature` - a list of features to enable when compiling the crate.
```

Comment:
Is this a comma-separated list?

Reply:
Ah, ignore this, I was thinking in the context of the command line, not the config. :)

Reply:
Ah....I do think this should be `features` (plural) though.